### PR TITLE
Update 13-dplyr.Rmd

### PR DESCRIPTION
**Pull request by potterzot**
Didn't install and load ggplot2 so was giving errors in the last section combining dplyr and ggplot2

The current lesson doesn't load ggplot2 so the output gives errors about ggplot2 not being loaded. These revisions install and load.

### CODE REVIEW

File: _episodes_rmd/13-dplyr.Rmd [hunk 1]
```diff
@@ -317,6 +317,18 @@ gdp_future_bycontinents_byyear_high_lifeExp <- gapminder %>%
 
 ## Combining `dplyr` and `ggplot2`
 
+First install and load ggplot2:
+
+```{r,eval=FALSE}
+install.packages('dplyr')
```
Comment (jcoliver) [inline _episodes_rmd/13-dplyr.Rmd]: Remove `install.packages('dplyr')` from PR

**Comment by jcoliver** [comment]
Thanks, @potterzot ! The lesson already includes the instructions for installing and loading the dplyr package (in the section called "The dplyr package"). However, you are right about ggplot - even the rendered html currently has all those errors! Would you mind revising the pull request to only have the instructions for installing and loading ggplot2 package? That is, remove the `install.packages("dplyr")` line that you added (it's line 323 or so).

**Comment by potterzot** [comment]
@jcoliver should be all set now, was making that PR on the fly and erroneously copied it from above. Thanks!

**Comment by jcoliver** [comment]
Rock on. Thanks, @potterzot .

**Pull request merged**